modules/ROOT/partials/couchbase-operator-2.7.1-release-notes.adoc

```diff
@@ -0,0 +1,39 @@
+[#release-271]
+== Release 2.7.1 (August 2025)
+
+Couchbase Operator 2.7.1 was released in August 2025.
+This maintenance release contains fixes to issues.
+
+[#fixed-issues-v271]
+== Fixed Issues
```

Comment:
This indent level is different to the others, and makes it show up in the table of contents on the right hand side - can it please be increased one level?